src/tests/components/exportable/FullCard.test.tsx

```diff
@@ -84,38 +84,44 @@ describe('FullCard', () => {
         it('renders without crashing', () => {
             expect(wrapper.find(FullCard).exists()).toBe(true);
         });
+        it('renders the links footer', () => {
+            expect(wrapper.find('div .c-links').exists()).toBe(true);
+        });
```

Comment:
Estaría bueno también testear el caso negativo, de que si `links` === `none`, no esté la clase `c-links`

Reply:
Está hecho! En la línea 104:

`        it('does not render the links footer', () => {
            mockCardOptions.links = "none";
            mountFullCard();
            expect(wrapper.find('div .c-links').exists()).toBe(false);
        });`